Description: Use sensible-editor instead of emacs
 and fix RPATH issue.

Forwarded: no
Author:  Hugo Vanwoerkom <hugovanwoerkom@yahoo.com>, Josue Abarca <jmaslibre@debian.org.gt>
Last-Update: 2010-01-22

--- a/Local.config
+++ b/Local.config
@@ -29,7 +29,7 @@
 XCOMM PRINTCAP = /etc/printcap:/etc/printcap.cups
 XCOMM #define PRINTCAP 
 
-EDITOR = emacs -fn 9x15 -cr green -ms red -bg lightyellow -fg black
+EDITOR = x-terminal-emulator -e sensible-editor
 POSTSCRIPT_VIEWER = gv
 EXTERN_VIEWER = display
 
@@ -147,7 +147,7 @@
 #if defined(__NetBSD__) || defined(__FreeBSD)
 XPM_LIB     = -L@XPMDIR@/lib -Wl,-R@XPMDIR@/lib -lXpm -lX11 -lm -lXmu -lXext
 #else
-XPM_LIB     = -L/usr/lib -Wl,-R/usr/lib -lXpm -lX11 -lm -lXmu -lXext
+XPM_LIB     = -L/usr/lib -lXpm -lX11 -lm -lXmu -lXext
 #endif
 
 XCOMM  Make sure this points to the location of the 'xpm.h' file.
